index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <meta name="author" content="Mavericks Balitaan" />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="container">
+      <header>
+        <div class="header">
+          <div class="headerText">Welcome</div>
+          <div class="menu">
+            <img src="./svg/hamburger.svg" alt="menu" />
+          </div>
```

Comment:
- [ ] Kindly use `nav` tag here instead of `div` tag, because this is your menu and you will put your links later, so it's more semantic to use `nav` tag.